go/cmd/vtbackup/vtbackup.go

```diff
@@ -203,6 +218,13 @@ func main() {
 	topoServer := topo.Open()
 	defer topoServer.Close()
 
+	// Initialize stats.
+	for phaseName, statuses := range phaseStatuses {
+		for _, status := range statuses {
+			phaseStatus.Set([]string{phaseName, status}, int64(0))
```

Comment:
is the explicit `int64` cast necessary here?

Reply:
Nope! Fixed.